tests/tango/test_base_device.py

```diff
@@ -390,3 +478,41 @@ async def test_signal_autofill(tango_test_device, auto_fill_signals):
     else:
         assert not hasattr(test_device, "ignored_attr")
         assert not hasattr(test_device, "readback")
+
+
+@pytest.mark.asyncio
```

Comment:
Tango under windows is currently tested, but the runners are quite weedy, so need a bigger timeout. Does something like this work?
```suggestion
@pytest.mark.asyncio
@pytest.mark.timeout(10.0)
```

Reply:
It seems as if this error occurs at the end of the last test in test_tango_transport. I was seeing the error on the last test, so I tried splitting that into smaller tests and I still see it at the end of the last newly defined test.

Reply:
The last 4 tests in test_tango_transport: `test_type_mismatch_justvalue`, `test_type_mismatch_array`, `test_type_mismatch_sequence`, and `test_type_mismatch_longstringarray`, seem to be the culprits. If I comment out all 4, the tests pass on windows. If I comment out any subset of them, it times out on the last one. Whether I include justvalue or all four, it times out after the last test.

Reply:
In this commit 1f4a3d45b45bc5b3a7cb34900664b9671e37483a you can see the windows tests all passed when I commented out those tests,

Reply:
Ahh, I see. I didn't look far enough back, I just check the last couple of logs. I have no idea why the process hangs, but I notice we're currently pinning Tango for some reason:
https://github.com/bluesky/ophyd-async/blob/5e6b1b932ac54c18f8aceb10111219e27fd0a33b/pyproject.toml#L34

If we remove the pin does it fix it?

Reply:
@coretl I am not sure what to do about the windows tests. They seem to fail in different ways each time I make a change. It doesn't just seem to be the tango tests that are failing. On main it looks like we've been having problems with Windows tests for a while now. Could we put this PR through before the windows issues are sorted?